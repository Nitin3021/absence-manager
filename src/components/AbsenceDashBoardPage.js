import React from 'react';
import AbsenceList from './AbsenceList';
import AbsenceListFilters from './AbsenceListFilters';

const AbsenceDashBoardPage = () => (
    <div>
        <AbsenceListFilters />
        <AbsenceList />
    </div>
);

export default AbsenceDashBoardPage;