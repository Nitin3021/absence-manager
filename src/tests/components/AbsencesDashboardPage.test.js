import React from 'react';
import { shallow } from 'enzyme';
import AbsenceDashBoardPage from '../../components/AbsenceDashBoardPage';

test('should render AbsenceDashBoardPage correctly', () => {
    const wrapper = shallow(<AbsenceDashBoardPage />);
    expect(wrapper).toMatchSnapshot();
});