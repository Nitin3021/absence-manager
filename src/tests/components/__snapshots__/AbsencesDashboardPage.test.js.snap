// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AbsenceDashBoardPage correctly 1`] = `
<div>
  <Connect(AbsenceListFilters) />
  <Connect(AbsenceList) />
</div>
`;
